src/modules/dashboard/components/SmiteDialog/SmiteDialog.tsx

```diff
@@ -126,6 +126,22 @@ const SmiteDialog = ({
     [totalReputationData],
   );
 
+  const { walletAddress: loggedInUserWalletAddress } = useLoggedInUser();
+
+  const selectedUser = useMemo(() => {
```

Comment:
I think this function could be converted to a util function/hook, as it may be needed in other dialogs. You could do that in this PR, otherwise, you could merge it, and then I convert it in #2770

Reply:
I agree with @ArmandoGraterol, as this should be a hook, since he's going to use it as well for the "Awards" modal.

The more hooks the better :)

But it's up to you guys where / who decides to implement it.

Reply:
done